src/modules/navigator/mission_block.cpp

```diff
@@ -452,13 +452,15 @@ MissionBlock::issue_command(const mission_item_s &item)
 		vcmd.param2 = item.params[1];
 		vcmd.param3 = item.params[2];
 		vcmd.param4 = item.params[3];
-		vcmd.param5 = item.params[4];
-		vcmd.param6 = item.params[5];
 
 		if (item.nav_cmd == NAV_CMD_DO_SET_ROI_LOCATION && item.altitude_is_relative) {
+			vcmd.param5 = item.lat;
+			vcmd.param6 = item.lon;
 			vcmd.param7 = item.params[6] + _navigator->get_home_position()->alt;
```

Comment:
So here we might as well use `item.alt` to be consistent, right?

Reply:
True, didn't realize that thanks to the `union` the two fields actually occupy the same bytes.

Reply:
Oh funny.

Reply:
:heavy_check_mark: